build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.2'
+        classpath 'com.android.tools.build:gradle:3.4.0'
```

Comment:
brave!! xD

Reply:
You are brave enough to be always in the latest version of Android Studio but you don't want to test @Parcelize, interesting... 🤔